Solidner/Views/Onboarding/OnboardingTitles.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OnboardingTitles.swift
+//  Solidner
+//
+//  Created by 황지우2 on 2023/11/12.
+//
+
+import SwiftUI
+
+struct OnboardingTitles: View {
+    let bigTitleColor = Color(#colorLiteral(red: 0.0834152922, green: 0.0834152922, blue: 0.0834152922, alpha: 1))
+    let smallTitleColor = Color(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)).opacity(0.6)
+    let leadingPadding = CGFloat(4)
+    let bigTitleFontSize = CGFloat(28)
+    let smallTitleFontSize = CGFloat(19)
+    let smallTitleTopPadding = CGFloat(16)
+    let centerSmallTitleTopPadding = CGFloat(10)
+    @State var alignmentCase = AlignmentCase.leading
+    @State var bigTitle = ""
+    @State var smallTitle = ""
+    @State var isSmallTitleExist = true
+    var body: some View {
+        switch alignmentCase {
+        case .leading :
+            leadingTitles()
+        case .center :
+            centerTitles()
+        }
+    }
+    
+    func leadingTitles () -> some View {
```

Comment:
```suggestion
    var leadingTitles: some View {
```
p5;
매개변수를 받지 않으므로 computed property를 활용해도 좋을듯